src/hls.js

```diff
@@ -51,6 +51,10 @@ export const attatchStream = media => {
         hls.startLoad()
         media.dispatchEvent(new CustomEvent('error', { detail: { networkState: HTMLMediaElement.NETWORK_EMPTY } }))
         break
+     case Hls.ErrorTypes.MEDIA_ERROR:
+        console.log("fatal media error encountered, try to recover");
```

Comment:
not sure if a console.log is the appropriate messaging event

Reply:
Me neither. Removing the line is definitely an option as it's supposedly self-healing. But it might be useful to see that this branch was taken as well.

Reply:
remove it :D